providers/fab/docs/changelog.rst

```diff
@@ -47,131 +47,33 @@ Breaking changes
 
 * ``Prepare FAB provider to set next version as major version (#43939)``
 * ``Remove deprecations from fab provider (#44198)``
+* ``Rename 'get_permitted_dag_ids' and 'filter_permitted_dag_ids' to 'get_authorized_dag_ids' and 'filter_authorized_dag_ids' (#47640)``
+* ``Set simple auth manager as default (#47691)``
 
```

Comment:
"Removed ``oauth_whitelists`` property from the security manager override" is no longer true because it was added back in #48968. Not sure if we can delete it or the scripts will not be happy?

Reply:
Done